### DISCUSSION TITLE
Move prebuilt option to OSS version to support other languages

### DISCUSSION THREAD

**Discussion by brumhard** [goreleaser/goreleaser#3986] [Ideas, Issue Triage, and General discussions]
Hi,

I love goreleaser for Go projects for obvious reasons. So I played around with goreleaser today to try to release a small rust binary with it (since there's no real competitor in the rust ecosystem I think).
With the skip option of builds and a few hacks (manually disabling all build related stuff and adding `extra_files` everywhere) I was able to at least get the binaries uploaded, a signed checksum as well as a nice changelog (see [here](https://github.com/brumhard/reinlinsen) for the used goreleaser file).

What I couldn't get working is the archives as well as linux packages since they are missing the builds/ the OSs defined in the build section which leads to errors.
Then I found the `prebuilt` option (https://goreleaser.com/customization/builds/#import-pre-built-binaries) which I think would resolve most of the hacks but sadly it's pro only.
That's why I wanted to propose that you consider making the `prebuilt` option part of the OSS release and with that dive into all the other language ecosystems and become the standard release tool across all projects.
Ofc all the build options like ko etc. don't really make sense for other languages but I still think goreleaser would provide enormous value to the OSS communities of other languages with that change.

Thanks for your nice work 🚀

**Comment by caarlos0** [comment]
Hey, thanks for the kind words, but for the time being I have no intention of merging that into the OSS version.

The OSS version already does a whole lot, for free... so much, in fact, that's a bit hard to find things to do only in the Pro version.

Once I have more pro-only features, maybe...

Thanks for the suggestion, though! Cheers!

**Reply by brumhard** [reply]
Yup already expected that and I can absolutely understand. 
Maybe it might be an option in the future to get more customers by getting into more ecosystems, but ofc I might also be the one guy who tries to stick to the tools I'm used to too much 😅 
Anyways, ty and keep up the great work 🙏🏻
